### PR TITLE
Add contacts menu integration

### PR DESCRIPTION
**Pull request by Ivansss**
- [ ] Start a call with the user (internal users only) selected in contacts menu.

### CODE REVIEW

**Comment by nickvergessen** [comment]
Works here, good to merge?

**Comment by Ivansss** [comment]
Works fine here as well 👍

File: lib/Controller/PageController.php [hunk 1]
```diff
@@ -89,10 +95,11 @@ public function __construct($appName,
 	 * @NoCSRFRequired
 	 *
 	 * @param string $token
+	 * @param string $callUser
 	 * @return TemplateResponse
-	 * @throws HintException
+	 * @throws HintException|RedirectResponse
```
Comment (ChristophWurst) [inline lib/Controller/PageController.php]: it *returns* the response 😉

**Comment by Ivansss** [comment]
Tooltips are only visible for the first action (the most to the left).

No videocall tooltip visible here. Only visible the email tooltip.
screen shot 2017-04-26 at 14 34 05

Videocall tooltip is visible here because there is only one action.
screen shot 2017-04-26 at 14 33 13

@ChristophWurst

**Comment by ChristophWurst** [comment]
@Ivansss now I see the problem. We have to add the tooltip to the second action too. Please create a ticket in the server repo and assign me. I'll take a look next week. Thanks :)

**Comment by jancborchardt** [comment]
@Ivansss the tooltip is fixed to correct capitalized »Video call« now, right? ;)

**Comment by Ivansss** [comment]
@jancborchardt yes :) it was changed here #303

**Pull request merged**